utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/schema/AvroSchemaBasedParameter.scala

```diff
@@ -26,11 +27,11 @@ object AvroSinkValueParameter {
   /*
     We extract editor form from Avro schema
    */
-  def apply(schema: Schema)(implicit nodeId: NodeId): ValidatedNel[ProcessCompilationError, SinkValueParameter] =
-    toSinkValueParameter(schema, paramName = None, defaultValue = None)
+  def apply(schema: Schema)(implicit nodeId: NodeId): ValidatedNel[ProcessCompilationError, SchemaBasedParameter] =
```

Comment:
Let's extract this `restrictedParamNames` field to method parameters and use an empty set in our testing mechanism - we don't need to check those parameters names there - it is now sink specific and should be moved to the the place where sink uses it.

Reply:
moved `restrictedParamNames` to method params and moved `restrictedParamNames` declarations to specific sinks.